### PR TITLE
Devsbom 062023

### PR DESCRIPTION
**Pull request by sumanthkb44**
PR Contains NPM, Python & Debian changes of importing multiple SBOM files along with UT & IT changes.

### CODE REVIEW

File: doc/UsageDoc/CA-Tool--SBOM-Vision.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## CA Tool
+The Continuous Clearing Tool scans and collects the 3rd party OSS components used in a NPM/NuGet/Maven/Debian project and uploads it to SW360 and Fossology by accepting respective project ID for license clearing.
+
```
Comment (karthika-g) [inline doc/UsageDoc/CA-Tool--SBOM-Vision.md:3]: Keep updating usage.md as well with the changes for python

File: doc/UsageDoc/CA-Tool--SBOM-Vision.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+## CA Tool
```
[the remark this answers is not in the document]
Comment (sumanthkb44) [reply doc/UsageDoc/CA-Tool--SBOM-Vision.md]: Changed

File: doc/UsageDoc/CA-Tool--SBOM-Vision.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+The Continuous Clearing Tool scans and collects the 3rd party OSS components used in a NPM/NuGet/Maven/Debian project and uploads it to SW360 and Fossology by accepting respective project ID for license clearing.
+
+## CA Tool SBOM
+BOM generated from the CA Tool will have all the dependencies included in the scanned package , component type, Source repo URL, dependency type etc..,
+
+##Features
```
[the remark this answers is not in the document]
Comment (sumanthkb44) [reply doc/UsageDoc/CA-Tool--SBOM-Vision.md]: Changed

File: src/LCT.APICommunications/PythonJfrogAPICommunication.cs [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        }
+        private static HttpClient GetHttpClient(ArtifactoryCredentials credentials)
+        {
+            HttpClient httpClient = new HttpClient();
+
+            httpClient.DefaultRequestHeaders.Add(ApiConstant.JFrog_API_Header, credentials.ApiKey);
+            httpClient.DefaultRequestHeaders.Add(ApiConstant.Email, credentials.Email);
+            return httpClient;
+        }
+        //public override async Task<HttpResponseMessage> CheckPackageAvailabilityInRepo(string repoName, string componentName, string componentVersion)
```
[the remark this answers is not in the document]
Comment (sumanthkb44) [reply src/LCT.APICommunications/PythonJfrogAPICommunication.cs]: Removed

File: src/LCT.APICommunications/PythonJfrogAPICommunication.cs [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        //    return await httpClient.GetAsync(url);
+        //}
+        public override async Task<HttpResponseMessage> GetApiKey()
+        {
+            HttpClient httpClient = GetHttpClient(ArtifactoryCredentials);
+            string url = $"{DomainName}/api/security/apiKey";
+            return await httpClient.GetAsync(url);
+        }
+
+        //public override async Task<HttpResponseMessage> CopyPackageFromRemoteRepo(UploadArgs uploadArgs, string destreponame)
```
[the remark this answers is not in the document]
Comment (sumanthkb44) [reply src/LCT.APICommunications/PythonJfrogAPICommunication.cs]: Removed

**Review by karthika-g — APPROVED** [review]
Approved.will need to add maven IT files

**Pull request merged**